.changeset/happy-carrots-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Fixed issue with `Table` row selection when the row has disabled sub rows.
```

Comment:
This changeset is insufficient. Fixed what issue?

Reply:
Updated changeset in this [commit](https://github.com/iTwin/iTwinUI/pull/2013/commits/997270c33fad447c71ac44fa8ad08053b02b8935).